kafka_consumer/tox.ini

```diff
@@ -0,0 +1,86 @@
+
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+    kafkaconsumer-{0_9_0_1,0_10_2_1,0_11_0_1}-{kafka,zk}
+    flake8
+
+[common]
+deps =
+    ../datadog_checks_base
+    -rrequirements-dev.txt
+commands =
+    pip install --require-hashes -r requirements.txt
+
+[testenv]
+platform = linux2|darwin
+
+[testenv:kafkaconsumer-0_9_0_1-kafka]
+deps = {[common]deps}
+setenv =
+    KAFKA_VERSION=0.9.0.1-1
+    KAFKA_OFFSETS_STORAGE=kafka
+passenv = DOCKER_HOST*
```

Comment:
We need `COMPOSE*`. https://github.com/DataDog/integrations-core/pull/1472

Also, let's put `passenv` only in `[testenv]`.